HS9_py2/setup.py

```diff
@@ -43,4 +44,6 @@
                 'heatsource9.ModelSetup',
                 'heatsource9.Dieties',
                 'heatsource9.Stream',
-                'heatsource9.Utils'])
+                'heatsource9.Utils'],
+      ext_modules = cythonize("heatsource9/Stream/*.pyx")

```

Comment:
Compile the Stream modules as native C classes to improve speed and compiler optimization.